frontend/src/components/Modal/RiderModalInfo.tsx

```diff
@@ -1,112 +1,181 @@
+              'Wheelchair'
+            </p>
+          )}
         </div>
-        <div className={cn(styles.gridR3, styles.gridC2)}>
+        <div className={cn(styles.gridR2, styles.gridCBig2)}>
+          <SRLabel htmlFor='address'>Address: </SRLabel>
           <Input
+            id="address"
             name="address"
             type="text"
             placeholder="Address"
-            ref={register({ required: true })}
+            ref={register({
+              required: true,
+              pattern: /^[a-zA-Z0-9\s,.'-]{3,}$/,
+            })}
           />
-          {errors.email && 'enter an address'}
+          {errors.address && (
+            <p className={styles.error}>Please enter an address</p>
+          )}
         </div>
-        <div className={cn(styles.gridR4, styles.gridC1)}>
-          <Label htmlFor="start">Start Date:</Label>
-          <Input
-            type="date"
-            name="start"
-            ref={register({ required: true })}
-          />
-        </div>
-        <div className={cn(styles.gridR4, styles.gridC2)}>
-          <Label htmlFor="end">End Date:</Label>
-          <Input
-            type="date"
-            name="end"
-            ref={register({ required: true })}
-          />
+        <div className={cn(styles.gridR3, styles.gridCAll)}>
+          <p>Duration</p>
+          <div className={styles.lastRow}>
+            <div>
+              <SRLabel htmlFor='joinDate'>Join Date: </SRLabel>
+              <Input
+                id='joinDate'
+                type="date"
+                name="joinDate"
+                ref={register({ required: true })}
+                className={styles.riderDate}
+              />
+              {errors.joinDate && (
+                <p className={styles.error}>Please enter a join date</p>
+              )}
+            </div>
+            <p className={styles.to}>to</p>
+            <div>
+              <SRLabel htmlFor='endDate'>End Date: </SRLabel>
+              <Input
+                type="endDate"
+                name="endDate"
+                ref={register({
+                  required: true,
+                  validate: (endDate) => {
+                    const joinDate = getValues("joinDate");
+                    return joinDate < endDate;
+                  },
+                })}
+                className={styles.riderDate}
+              />
+              {errors.end?.type === "required" && (
+                <p className={styles.error}>Please enter an end date</p>
+              )}
+              {errors.end?.type === "validate" && (
+                <p className={styles.error}>Invalid end time</p>
+              )}
+            </div>
+          </div>
         </div>
       </div>
-      <Button type="submit">Add a Rider</Button>
+      {/* TODO: styling */}
+      <div className={styles.buttonContainer}>
+        <Button className={styles.cancel} outline={true} onClick={() => cancel()}>Cancel</Button>
```

Comment:
Right now, if you press enter regardless of where you are on the modal, it will press cancel because the default button type is "submit". Add type = "button" to this button to fix this. That also does bring up the question for the need for the cancel button given the x at the top of the modal but that's a design question, not a me question.